src/Serializer/AbstractItemNormalizer.php

```diff
@@ -249,6 +249,11 @@ public function denormalize($data, $class, $format = null, array $context = [])
             return $item;
         }
 
+        // Filter out input attributes that aren't allowed
+        $data = array_filter($data, function (string $attribute) use ($context) {
+            return $this->canAccessAttribute($context[self::OBJECT_TO_POPULATE] ?? null, $attribute, $context);
+        }, \ARRAY_FILTER_USE_KEY);
```

Comment:
Is this change necessary? Could you denormalize and update / create a secured field without the right to do so before?